.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+---
+name: OpenWISP Users CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-18.04
```

Comment:
Shouldn't we run this on ubuntu-20.04

Reply:
@devkapilbansal Yes, let me change it quickly.